superset/examples/deck.py

```diff
@@ -387,6 +387,7 @@ def load_deck_dash() -> None:  # pylint: disable=too-many-statements
         "stroked": False,
         "extruded": True,
         "multiplier": 0.1,
+        "line_width": 10,
```

Comment:
Shouldn't we also add `"line_width_unit": "meters"` here, as the default unit is now being changed?

Reply:
Great catch!